sos/report/plugins/discovery.py

```diff
@@ -29,6 +29,7 @@ def setup(self):
 
         self.add_container_logs([
             'discovery',
-            'dsc-db'
+            'dsc-db',
+            'discovery-toolbox'
         ])
```

Comment:
Maybe to prevent doing changes on 2 places, we can have here

    self.add_container_logs(list(self.containers))

? But I am OK with the current change.